### ISSUE TITLE
The selection of subdataset of tonic.dataset

### ISSUE THREAD

**Issue by mosfe** [neuromorphs/tonic#247]
I have used 'tonic.dataset.SSC(download_dir, split='train',transform=transform)' to get the full dataset of SSC dataset, and it contains 35 classes. However. I only want to train some classes, such as [0,1,2,3,4,5,6]. How could I get the subdataset qucikly, I have tried it using the for loop method, but it seems a little stupid. What should I do?
Hope for your reply

**Comment by biphasic** [comment]
Calculate the indices of the samples that you want and store them in a numpy file. Then use Pytorch's [Subset](https://pytorch.org/docs/stable/data.html#torch.utils.data.Subset)

**Comment by biphasic** [comment]
This behaviour will change in Tonic 2.0 when we support to iterable dataset such as supported in torchdata! But for now manually looping over the labels is the simplest option

**Comment by biphasic** [comment]
Actually that's already possible like so:
```
import tonic
from torchdata.datapipes.iter import IterableWrapper

dp = IterableWrapper(tonic.datasets.NMNIST('data'))
dp = dp.filter(lambda data: data[1]==0) # only get samples with label 0
```
